definitions/divorce/json/CaseField.json

```diff
@@ -3395,5 +3395,14 @@
     "Label": "Additional information",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "23/09/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ApplyForDecreeAbsolute",
+    "Label": "Once the court has checked that there are no outstanding applications or other matters which need to be resolved first, the decree nisi made on {dyinamic date} will be made absolute and the mariage between {dynamic data} and {dynamic data} will be ended",
```

Comment:
Dynamic fields to use would be: `${DNApprovalDate}`, `${D8DerivedPetitionerCurrentFullName}` and `${D8DerivedRespondentCurrentName}`